pom.xml

```diff
@@ -304,7 +304,7 @@
                         <!-- enforce that tests should be independent -->
                         <runOrder>random</runOrder>
                         <!-- re-run flaky tests once -->
```

Comment:
comment no longer quite right, can probably just explain why it would be a variable